target/linux/x86/geode/config-6.6

```diff
@@ -35,43 +35,36 @@ CONFIG_ACPI_THERMAL=y
 # CONFIG_ACPI_WMI is not set
 # CONFIG_ADV_SWBUTTON is not set
 CONFIG_ALIX=y
-# CONFIG_AMD_PMC is not set
 # CONFIG_AMD_PMF is not set
-CONFIG_ARCH_MIGHT_HAVE_ACPI_PDC=y
 # CONFIG_ASUS_TF103C_DOCK is not set
+CONFIG_ARCH_MIGHT_HAVE_ACPI_PDC=y
 # CONFIG_ATA_PIIX is not set
-CONFIG_CRYPTO_LIB_BLAKE2S_GENERIC=y
 CONFIG_CS5535_CLOCK_EVENT_SRC=y
 CONFIG_CS5535_MFGPT=y
 CONFIG_CS5535_MFGPT_DEFAULT_IRQ=7
 # CONFIG_CS89x0_ISA is not set
 CONFIG_DMA_ACPI=y
 # CONFIG_EL3 is not set
+CONFIG_FUNCTION_ALIGNMENT=16
+CONFIG_FUNCTION_ALIGNMENT_16B=y
+CONFIG_FUNCTION_PADDING_BYTES=16
+CONFIG_FUNCTION_PADDING_CFI=11
```

Comment:
This surprises me.  I could believe Geode likes 16-byte aligned functions, but as a late-model 32-bit platform I would expect 4-byte or 8-byte alignment.  I don't have the 6.6 source in front of me nor I am particularly familiar with the Geode processors.